### ISSUE TITLE
Initiators: Research and prototype CMR trigger options (PUSH)

### ISSUE THREAD

**Issue by galenatjpl** [unity-sds/unity-cs#455]

**Comment by pymonger** [comment]
Currently subscriptions are broken in CMR. I've submitted a ticket into CMR which can be tracked at:

https://bugs.earthdata.nasa.gov/browse/EDSC-4204

![Image](https://github.com/user-attachments/assets/186d62a6-cef8-4bb3-8276-b045e8de4da5)

**Comment by galenatjpl** [comment]
Currently BLOCKED since CMR is working on a JIRA ticket (see above) that blocks this.

**Comment by pymonger** [comment]
@galenatjpl: this seems to be unblocked now but should we put this in the backlog due to the SPS work?

**Comment by galenatjpl** [comment]
@pymonger @LucaCinquini , yeah I think this can go into the SPS backlog.  It can be up to you all how to prioritize this work. It's a pretty cool feature though, and I think multiple missions would benefit from it, so it might be a good "reference" initiator/trigger to implement anyway.
